tooling/v8-snapshot/src/doctor/snapshot-doctor.ts

```diff
@@ -313,7 +309,7 @@ export class SnapshotDoctor {
     meta: Metadata
   }> {
     // 1. Generate the initial bundle not deferring anything yet
-    let { warnings, meta, bundle } = await this._createScript()
+    let { meta } = await this._createScript()
```

Comment:
Looks like this can be a `const`?

Reply:
Good call [cypress-io/cypress@`ec34a53` (#23732)](https://github.com/cypress-io/cypress/pull/23732/commits/ec34a537a4879edcfbc0409b0f81f16a94e97ee9)